testregistry.textproto

```diff
@@ -434,6 +434,12 @@ test: {
   readme: ""
   exec: " "
 }
+test: {
+  id: "PF-1.7"
```

Comment:
@dplore `PF`?

Reply:
PF == policy forwarding types of tests.  This PR breaks out a portion of a gribi test which only manipulates policy-forwarding paths, so it makes sense of the moved subtest to be given a new ID in the "PF" category.